src/ExContext.elm

```diff
@@ -206,6 +208,22 @@ getType =
     getFromContext .types
 
 
+{-| Get's arity of the function in the module
+-}
+getArity : String -> String -> Context -> Maybe Int
+getArity m fn ctx =
+    ctx.modules
+        |> Dict.get m
+        |> Maybe.map (.definitions)
+        |> Maybe.andThen (Dict.get fn)
+        |> Maybe.map (.arity)
```

Comment:
According to Elm's argument order convention it should be 
```
getArity ctx m fn
```
as the function is the target

Reply:
Also there is no need for parens around getter functions